rules/bigquery_rules.yaml

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# The BigQuery scanner rules are blacklists.
+# BigQuery scanner rules default to mode 'blacklist'.
 rules:
   - name: BigQuery rule to search for public datasets
     dataset_id: '*'
```

Comment:
Add "mode: blacklist" to these default rules in case it becomes a required field later.

Reply:
Done.